src/app/home-page/widget/starred-box/starred-box.component.html

```diff
@@ -0,0 +1,188 @@
+    <mat-card-header fxLayout="row">
+      <mat-card-title>Starred</mat-card-title>
+    </mat-card-header>
+    <div id="total-starred" fxLayout="row" fxLayoutGap="1rem" class="size-small pb-3">
+      <div [ngPlural]="totalStarredWorkflows" id="starred-workflows" fxLayout="row" fxLayoutAlign="start center">
+        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" /> 0 Workflows </ng-template>
+        <ng-template ngPluralCase="other">
+          <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" /> {{ totalStarredWorkflows }} Workflows</ng-template
+        >
+      </div>
+      <div [ngPlural]="totalStarredTools" id="starred-tools" fxLayout="row" fxLayoutAlign="start center">
+        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" /> 0 Tools </ng-template>
+        <ng-template ngPluralCase="other">
+          <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" /> {{ totalStarredTools }} Tools</ng-template
+        >
+      </div>
+      <div [ngPlural]="totalStarredServices" id="starred-services" fxLayout="row" fxLayoutAlign="start center">
+        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" /> 0 Services </ng-template>
+        <ng-template ngPluralCase="other">
+          <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" /> {{ totalStarredServices }} Services</ng-template
+        >
+      </div>
+      <div [ngPlural]="totalStarredOrganizations" id="starred-orgs" fxLayout="row" fxLayoutAlign="start center">
+        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" /> 0 Organizations </ng-template>
+        <ng-template ngPluralCase="other">
+          <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" /> {{ totalStarredOrganizations }} Organizations</ng-template
+        >
+      </div>
+    </div>
+  </div>
+  <mat-divider></mat-divider>
+  <mat-card-content class="p-3 h-100">
+    <div *ngIf="events.length !== 0 || isLoading; else noEvents">
+      <h5 class="weight-bold">Recent Activity</h5>
+      <div class="size-small widget-list-items" *ngFor="let event of events" fxLayout="row" fxLayoutAlign="space-between center">
+        <div fxLayout="row" fxLayoutAlign="start center">
+          <img src="{{ event?.initiatorUser?.avatarUrl }}" alt="User avatar" />
+          <div [ngSwitch]="event.type">
+            <div *ngSwitchCase="EventType.ADDVERSIONTOENTRY" class="my-3">
+              <div class="list-events">
+                <strong>{{ event.initiatorUser?.username }} </strong> created the {{ event.version?.referenceType | lowercase }}
+                <strong>{{ event.version?.name }}</strong> in {{ event?.tool || event?.apptool ? 'tool' : 'workflow' }}
+                <a
+                  [routerLink]="
+                    event?.apptool
```

Comment:
I think this would be better with a pipe, and line 54 as well. Both because it would be easier to read and test, but also because the logic is duplicated below. But it can be a followup ticket.

Reply:
I will add this to the list of followup tickets I make before leaving :D